<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ASHWIN" failures="0" tests="2" name="tests.Appointments_TC__004" time="0.018" errors="1" timestamp="2023-11-07T07:33:43 IST" skipped="1">
  <testcase classname="tests.Appointments_TC__004" name="openAndroidApk" time="0.018">
    <error message="java.io.FileNotFoundException: C:\Users\vashw\Desktop\Selenium\MedopractMobileAppAutomation\src\test\resources\log2.txt (The system cannot find the path specified)" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: C:\Users\vashw\Desktop\Selenium\MedopractMobileAppAutomation\src\test\resources\log2.txt (The system cannot find the path specified)
at org.openqa.selenium.remote.service.DriverService$Builder.getLogOutput(DriverService.java:464)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:509)
at io.appium.java_client.service.local.AppiumDriverLocalService.buildService(AppiumDriverLocalService.java:92)
at genericPages.CommonMethod.setInstance(CommonMethod.java:87)
at genericPages.CommonMethod.getInstance(CommonMethod.java:92)
at genericPages.CommonMethod.initializeDriver(CommonMethod.java:105)
at tests.Appointments_TC__004.openAndroidApk(Appointments_TC__004.java:31)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.FileNotFoundException: C:\Users\vashw\Desktop\Selenium\MedopractMobileAppAutomation\src\test\resources\log2.txt (The system cannot find the path specified)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:295)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:236)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:185)
at org.openqa.selenium.remote.service.DriverService$Builder.getLogOutput(DriverService.java:462)
... 30 more
]]>
    </error>
  </testcase> <!-- openAndroidApk -->
  <testcase classname="tests.Appointments_TC__004" name="bookAppointmentInHomePage" time="0.000">
    <skipped/>
  </testcase> <!-- bookAppointmentInHomePage -->
</testsuite> <!-- tests.Appointments_TC__004 -->
